Mapper.md

```diff
@@ -55,7 +55,7 @@ fun <From, To, K> Mapper<From, To>.map(value: Map<K, From>): Map<K, To> { ... }
 ## Default implementations
 
 - `VoidMapper`: Empty implementation that returns en error.
-- `BlankMapper<T>`: Blank mapper that returns the same object.
+- `IdentityMapper<T>`:  Mapper that returns the same object.
```

Comment:
I think IdentityMapper is a much better name, so no real comments here. 

Nevertheless, identity transformations made me think of times where we need to map between two types that have exactly the same property signatures. This can be achieved with _EncodableToDecodableMapper_, which is referred to as a Swift exclusive implementation.

Would it make sense to have the concept of a mirror mapper? Which could exist as a generic (and not swift-specific) mapper that creates a second type copying the names and types of properties of the first object. In Swift this mirror mapper would be the current _EncodableToDecodableMapper_ class, when no mappings are passed as parameters. But having a default MirrorMapper could unify the concept across implementations.

Reply:
Great @newboadki, so the consensus on the mapper rename is to call it `IdentityMapper`, as proposed.

Regarding the `MirrorMapper` you mention, this is exactly the `EncodableToDecodableMapper`. Would you rename it? I think is better to leave it more explicit, but you can convince me if you think is better not to.

Reply:
I think EncodableToDecodableMapper is a good name and if we can have the same name in Kotlin then that's it.

Yes MirrorMapper would be a particular case of EncodableToDecodableMapper, but thought that might be easier to implement in both platforms. Not a big deal.

Reply:
ok!